pyproject.toml

```diff
@@ -5,7 +5,7 @@ authors = [{ name = "Yifei Kong", email = "kong@yifei.me" }]
 description = "libcurl ffi bindings for Python, with impersonation support."
 license = { file = "LICENSE" }
 dependencies = [
-    "cffi>=1.12.0",
+    "cffi>=1.16.0",
```

Comment:
I prefer to keep the dependency version as low as possible if there are no serious bugs or features only available in new versions. The `cffi` lib is such a common dependency for many projects, see [here](https://github.com/python-cffi/cffi/network/dependents), raising it may cause conflicting situations for our users.